### PR TITLE
Rados tier-3 tests: Mondb trimming & scrubing scenarios.

### PR DESCRIPTION
**Pull request by pdhiran**
Signed-off-by: Pawan Dhiran <pdhiran@redhat.com>

Pass logs for new suite: http://magna002.ceph.redhat.com/ceph-qe-logs/pawan/cephci-run-UU5FTQ/

### CODE REVIEW

**Comment by pdhiran** [comment]
@SrinivasaBharath , Please have a look at the scrub tests pipeline scripts for any changes if needed.

File: pipeline/scripts/5/0/tier-3/test-rados-scrub-scenarios.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#! /bin/sh
+echo "Beginning Ceph RADOS regression testing for Scrubbing scenarios"
+
+random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
+instance_name="ci-${random_string}"
+platform="rhel-8"
+rhbuild="5.0"
+test_suite="suites/pacific/rados/tier_3_rados_scrub.yaml"
```
Comment (psathyan) [inline pipeline/scripts/5/0/tier-3/test-rados-scrub-scenarios.sh:8]: Could we rename this suite file to `tier-3_rados_test-scrubbing.yaml`?

This allows us to have a proper consolidated report at the end.

File: pipeline/scripts/5/0/tier-3/test-rados-mondb-trimming.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
+#! /bin/sh
+echo "Beginning Ceph RADOS regression testing for Mon DB trimming"
+
+random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
+instance_name="ci-${random_string}"
+platform="rhel-8"
+rhbuild="5.0"
+test_suite="suites/pacific/rados/tier-3_rados_mondb_trimming.yaml"
```
Comment (psathyan) [inline pipeline/scripts/5/0/tier-3/test-rados-mondb-trimming.sh:8]: Would it make sense to rename the file as `tier-3_raods_test-mon-db-trimming.yaml`?

File: suites/pacific/rados/tier-3_rados_mondb_trimming.yaml [hunk 3]
```diff
@@ -0,0 +1,173 @@
+# Suite contains tests related to mon db trimmings
```
Comment (psathyan) [inline suites/pacific/rados/tier-3_rados_mondb_trimming.yaml:1]: Can we rename this file to `tier-3_rados_test-mon-db-trimming.yaml`?

File: suites/pacific/rados/tier-3_rados_mondb_trimming.yaml [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              base_cmd_args:
+                verbose: true
+              args:
+                custom_repo: "cdn"
+                registry-url: registry.redhat.io
```
Comment (psathyan) [inline suites/pacific/rados/tier-3_rados_mondb_trimming.yaml:26]: Let's avoid having the registry url hardcoded... in IBM-C we use a different image registry.

File: pipeline/scripts/5/0/tier-3/test-rados-scrub-scenarios.sh (hunk 1 above)
Comment (pdhiran) [reply pipeline/scripts/5/0/tier-3/test-rados-scrub-scenarios.sh]: Renamed the suite file

File: pipeline/scripts/5/0/tier-3/test-rados-mondb-trimming.sh (hunk 2 above)
Comment (pdhiran) [reply pipeline/scripts/5/0/tier-3/test-rados-mondb-trimming.sh]: Renamed the suite file

File: suites/pacific/rados/tier-3_rados_mondb_trimming.yaml (hunk 4 above)
Comment (pdhiran) [reply suites/pacific/rados/tier-3_rados_mondb_trimming.yaml]: removed the registry link

File: suites/pacific/rados/tier-3_rados_mondb_trimming.yaml (hunk 3 above)
Comment (pdhiran) [reply suites/pacific/rados/tier-3_rados_mondb_trimming.yaml]: renamed the suite file

**Review by psathyan — APPROVED** [review]

**Pull request merged**